src/index.ts

```diff
@@ -53,5 +53,6 @@ export {
   ProcessPayload,
   DistinctivePropertiesPayload,
   BotDetectionResult,
-  BotKind,
+  BotKindType as BotKind,
+  BotKind as BotKindEnum,
```

Comment:
Having the object in the `BotKind` export is a more compatible solution (at least the JS user code won't break). I think it's ok to break the TypeScript compatibility a bit.

Reply:
It turns out removing the enum can be done without breaking changes:

```ts
export const BotKind = {
  Awesomium: 'awesomium',
  Cef: 'cef',
  // ...
} as const;
export type BotKind = typeof Fruit[keyof typeof BotKind];
```

I've checked that importing and re-exporting such a combined identifier works.